Sources/CodeEditSourceEditor/Extensions/TextView+/TextView+TextFormation.swift

```diff
@@ -42,10 +42,8 @@ extension TextView: TextInterface {
 
         delegate?.textView(self, willReplaceContentsIn: mutation.range, with: mutation.string)
 
-        layoutManager.beginTransaction()
```

Comment:
These fixes are required because of changes in this [PR](https://github.com/CodeEditApp/CodeEditTextView/pull/82)

@thecoolwinter once you get a PR up to update the `CodeEditTextView` version. I can rebase those changes into this branch.